scripts/install_environment.sh

```diff
@@ -160,6 +160,11 @@ function config_chronyd() {
   sudo firewall-cmd --zone=libvirt --add-port=123/udp
 }
 
+function deploy_reverse_proxy() {
```

Comment:
i don't think we need to setup ha proxy on create environment, i think it must be deployed only if needed

Reply:
from my point of view, run it from test infra when you need it